eng/Versions.props

```diff
@@ -13,34 +13,35 @@
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
     <AssemblyVersion>1.0.0.0</AssemblyVersion>
     <!-- .NET Runtime product dependencies -->
-    <MicrosoftBclAsyncInterfacesVersion>6.0.0</MicrosoftBclAsyncInterfacesVersion>
-    <MicrosoftExtensionsVersion>2.1.0</MicrosoftExtensionsVersion>
-    <MicrosoftExtensionsDependencyInjectionVersion>6.0.0</MicrosoftExtensionsDependencyInjectionVersion>
+    <MicrosoftBclAsyncInterfacesVersion>8.0.0</MicrosoftBclAsyncInterfacesVersion>
+    <MicrosoftExtensionsVersion>8.0.0</MicrosoftExtensionsVersion>
```

Comment:
We may want to break this one down too.

Reply:
Seems like it uses similar set of packages: https://github.com/dotnet/machinelearning/blob/5f5a199a5a97f17d8aa888765019f2344443a6a0/src/Microsoft.Extensions.ML/Microsoft.Extensions.ML.csproj#L10-L13

Reply:
Fixed.